apps/website/src/components/giveaway/GiveawayEntryConsentFieldset.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Link from "next/link";
+
+import { Fieldset } from "../shared/form/Fieldset";
+import { CheckboxField } from "../shared/form/CheckboxField";
+
+export function GiveawayEntryConsentFieldset() {
+  return (
+    <Fieldset legend="Data processing">
+      <CheckboxField isRequired={true}>
+        I have read and accept the{" "}
+        <Link className="underline" href="/privacy-policy" target="_blank">
```

Comment:
Is it worth using the content/Link component here with the external flag, to give the visual indicator this'll open in a new tab?